docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -10,90 +10,82 @@ keywords: changelog, table, materialize
 
 ## Context
 
-You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog stream into a table. This is broadly called *materializing* a changelog stream into a table.
+You have a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog into a table. This is broadly called *materializing* a changelog into a table.
 
 ## In action
 
 ```sql
 CREATE TABLE t1 AS
-    SELECT k,
-           LATEST_BY_OFFSET(v1) AS v1,
-           LATEST_BY_OFFSET(v2) AS v2,
-           LATEST_BY_OFFSET(v3) AS v3
-    FROM s1
-    GROUP BY k
+    SELECT *
+    FROM changelog
```

Comment:
If `changelog` is a STREAM, the updated query is invalid as the result is a `STREAM` and thus you cannot say `CREATE TABLE`.